tests/NLog.UnitTests/UnitTestApp.xaml.cs

```diff
@@ -1,4 +1,4 @@
-﻿#if NETSTANDARD && !NETSTANDARD1_3
+﻿#if NETSTANDARD && UWP10
```

Comment:
Well the code doesn't compile, unless having some UWP libraries. Didn't want to investigate, just wanted to test without NETSTANDARD1_3.